packages/@react-aria/dnd/src/useClipboard.ts

```diff
@@ -64,8 +66,10 @@ function addGlobalEventListener(event, fn) {
  * data types, and integrates with the operating system native clipboard.
  */
 export function useClipboard(options: ClipboardProps): ClipboardResult {
+  let {isDisabled} = options;
   let isFocusedRef = useRef(false);
   let {focusProps} = useFocus({
+    isDisabled,
```

Comment:
Question for the team: Should focus events be disabled here?

Reply:
I think it should always be enabled, otherwise you could end up in a scenario where the `isDisabled` prop is toggled, but focus occurred while disabled, then the isFocusedRef wouldn't be correct.

We're not sending any focus callbacks into `useClipboard`, so it won't be a problem of calling `onFocus` or the others anywhere outside.